opentelemetry-sdk/src/opentelemetry/sdk/logs/__init__.py

```diff
@@ -121,6 +165,10 @@ def __init__(
         self._resource = resource
         self._instrumentation_info = instrumentation_info
 
+    @property
```

Comment:
Good catch!

Reply:
Shouldn't this be on the `provider` level?

Reply:
Yes, this should be on provider as well.